analysis_options.yaml

```diff
@@ -29,6 +29,7 @@ linter:
   rules:
     avoid_relative_lib_imports: false
     package_api_docs: true
+    dangling_library_doc_comments: false
```

Comment:
I prefer that we don't disable recommended lints, but let us keep it for now, and fix the code later.